lua/wikis/chess/ratings_storage.lua

```diff
@@ -0,0 +1,56 @@
+local Json = require('Module:Json')
```

Comment:
imho could be integrated into Ratings.lua
(header missing yet again)
(needs some clean up too)